index.bs

```diff
@@ -87,13 +91,16 @@ This API provide high-end tools access to the same underlying data tables that b
 The API should:
 
 * Provide efficient enumeration of all local fonts without blocking the main thread
+* Ensure UAs are free to return anything they like. If a browser implementation prefers, they may choose to only provide a set of default fonts built into the browser.
 * Be available from Workers
 * Allow multiple levels of privacy preservation; e.g., full access for "trusted" sites and degraded access for untrusted scenarios
 * Reflect local font access state in the Permissions API
 * Restrict access to local font data to Secure Contexts
 * Provide unique identification of families and instances (variants like "bold" and "italic"), including PostScript names
-* Provide access to all browser-allowed font tables (may vary per browser)
+* Shield applications from unnecessary complexity by requiring that browser implementations produce valid OpenType data in the returned data
```

Comment:
We're not actually requiring/implementing this, are we? i.e. in Chrome we're not relying on OTS?

Reply:
Correct, we're not relying on OTS. That said I want to express that the byte data should be valid SFNT.
WDYT of switching`OpenType` for `SFNT` in this sentence?